pkg/watcherapplier/statefulset.go

```diff
@@ -0,0 +1,150 @@
+package watcherapplier
+
+import (
+	"github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	watcherv1beta1 "github.com/openstack-k8s-operators/watcher-operator/api/v1beta1"
+	watcher "github.com/openstack-k8s-operators/watcher-operator/pkg/watcher"
+	"k8s.io/utils/ptr"
+)
+
+const (
+	// ServiceCommand -
+	ServiceCommand = "/usr/local/bin/kolla_start"
+
+	// ComponentName -
+	ComponentName = watcher.ServiceName + "-applier"
+)
+
+// StatefulSet - returns the StatefulSet definition for the watcher-applier service
+func StatefulSet(
+	instance *watcherv1beta1.WatcherApplier,
+	configHash string,
+	labels map[string]string,
+) *appsv1.StatefulSet {
+	var config0644AccessMode int32 = 0644
```

Comment:
maybe that can go in constants.go too (I will re-use both it in my very similar StatefulSet function for DecisionEngine)

Reply:
agree, I can provide that in a follow up, or you could also provide that in your PR?

Reply:
ack sure i am duplicating in mine for now and once this merges can rebase